src/cogent3/core/sequence.py

```diff
@@ -2017,26 +2009,29 @@ def absolute_position(self, rel_index: int, include_boundary=False):
 
         # add offset and handle reversed views, now absolute relative to annotation coordinates
         offset = self.offset
-        if self.reverse:
+        if self.is_reversed:
             abs_index = offset + self.seq_len + seq_index + 1
         else:
             abs_index = offset + seq_index
 
         return abs_index
 
     def relative_position(self, abs_index, stop=False):
-        """converts an index relative to annotation coordinates to be with respect to the current sequence view
+        """converts an index relative to annotation coordinates to be with
```

Comment:
drop the reference to annotations. This is about converting an index on the original "python sequence" into an index on this "view"